tests/test_cli.sh

```diff
@@ -15,6 +15,22 @@ function setup_tests {
     # explicitly enable sshd for live-iso b/c it is disabled by default
     # due to security concerns (no root password required)
     sed -i.orig 's/^services.*/services --disabled="network" --enabled="NetworkManager,sshd"/' $share_dir/composer/live-iso.ks
+
+    # Make the live-iso boot more quickly (isolinux.cfg)
+    for cfg in "$share_dir"/templates.d/99-generic/live/config_files/*/isolinux.cfg; do
+        sed -i.orig 's/^timeout.*/timeout 20/' "$cfg"
+    done
+
+    # Make the live-iso boot more quickly (grub.cfg)
+    for cfg in "$share_dir"/templates.d/99-generic/live/config_files/*/grub.conf; do
+        sed -i.orig 's/^timeout.*/timeout 2/' "$cfg"
+    done
+
+    # Make the live-iso boot more quickly (grub2-efi.cfg)
+    for cfg in "$share_dir"/templates.d/99-generic/live/config_files/*/grub2-efi.cfg; do
+        sed -i.orig 's/^set timeout.*/set timeout=2/' "$cfg"
+    done
```

Comment:
I don't like the fact that we're not restoring the changes in `teardown_tests`. They are harmless but still better restore back to what was there before. You don't know if the same system won't be used for some manual testing afterwards.